msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.Common.targets

```diff
@@ -512,7 +538,8 @@ Copyright (C) 2014 Xamarin. All rights reserved.
 	<Target Name="_CompileInterfaceDefinitions" DependsOnTargets="_DetectAppManifest;_DetectSdkLocations;_CoreCompileInterfaceDefinitions" />
 
 	<Target Name="_CoreCompileInterfaceDefinitions">
-		<IBTool
+    <IBTool
+      SessionId="$(BuildSessionId)"
 			ToolExe="$(IBToolExe)"
 			ToolPath="$(IBToolPath)"
 			AppManifest="$(_AppManifest)"
```

Comment:
Looks like something happened with the indentation in you changes, could it be that we use \t rather than spaces?